From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: HNO3Miracle <hno3miracle@outlook.com>
Date: Sat, 23 May 2026 08:52:00 +0800
Subject: [PATCH] server/sotw: use constant status.Errorf format

Go 1.25 vet rejects non-constant format strings passed to Errorf.
---
 pkg/server/sotw/v3/ads.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pkg/server/sotw/v3/ads.go b/pkg/server/sotw/v3/ads.go
--- a/pkg/server/sotw/v3/ads.go
+++ b/pkg/server/sotw/v3/ads.go
@@ -49,7 +49,7 @@ func (s *server) processADS(sw *stream.Stream, reqCh <-chan *discovery.Discovery
 		// We only watch the multiplexed channel since we don't use per watch channels.
 		case res := <-respChan:
 			if err := sw.send(res); err != nil {
-				return status.Errorf(codes.Unavailable, err.Error())
+				return status.Errorf(codes.Unavailable, "%s", err.Error())
 			}
 		case req, ok := <-reqCh:
 			// Input stream ended or failed.
-- 
2.51.0

